app/models/publication.rb

```diff
@@ -31,6 +32,17 @@ class Publication < Publicationesque
   scope :non_statistical_publications, -> { where("publication_type_id NOT IN (?)", PublicationType.statistical.map(&:id)) }
   scope :corporate_publications, -> { where(publication_type_id: PublicationType::CorporateReport.id) }
 
+  def statistics_announcement_exists?
+    statistics_announcement.present? || statistics_announcement_id.present?
```

Comment:
Do we know we need both conditions? Wouldn't just the `statistics_announcement.present?` be enough?